### PR TITLE
Fixes #1867

### LINKED ISSUE #1867: pypodman commit: options "--change", "-f" and "--message" don't take effect
**kind bug**

**Description**
After fix #1743 , `pypodman commit` can run without any error and a commited image can be created. But options `--change`, `-f docker` and `--message` don't take effect.

**Steps to reproduce the issue:**
On Remote:
```
root@rhel8-1 ~]# podman ps -a
CONTAINER ID   IMAGE                             COMMAND     CREATED          STATUS                     PORTS   NAMES         IS INFRA
afacc34a5e9a   docker.io/library/fedora:latest   /bin/bash   13 seconds ago   Exited (0) 4 seconds ago           test-commit   false
```

On Host:
1. `~/.local/bin/pypodman -l root --host 192.168.122.213 commit --change ENTRYPOINT=/bin/sh --change LABEL=blue=image --message "commit test msg" -f docker test-commit image-commited`

2. `~/.local/bin/pypodman -l root --host 192.168.122.213 inspect image-commited`

**Describe the results you received:**
```
[
 { ......
   "Comment": "",
   ......
    "ContainerConfig": {
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "TERM=xterm",
                "HOSTNAME=",
                "container=podman",
                "FBR=f29",
                "DISTTAG=f29container",
                "FGC=f29"
            ],
            "Cmd": [
                "/bin/bash"
            ],
            "WorkingDir": "/"
        },
   ......
  "Labels": null,
   ......
 }
]
```

**Describe the results you expected:**
I did the same thing with `podman commit --change ENTRYPOINT=/bin/sh --change LABEL=blue=image --message "commit test msg" -f docker test-commit image-commited1` **on Remote**:
```
{
  .......
  "comment": "commit test msg",
  .......
  "containerconfig": {
    "env": [
      "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
      "TERM=xterm",
      "HOSTNAME=",
      "container=podman",
      "FBR=f29",
      "DISTTAG=f29container",
      "FGC=f29"
    ],
    "entrypoint": [
      "/bin/sh"
    ],
    "cmd": [
      "/bin/bash"
    ],
    "workingdir": "/",
    "labels": {
      "blue": "image"
    }
  },
  .......
  "labels": {
    "blue": "image"
  },
  .......
}
```
**Additional information you deem important (e.g. issue happens only occasionally):**
Besides, on host if I execute the `pypodman` command without `-f docker`, the image should not be created and a error message should be printed. But the fact is above wrong image can be still created.
If I do this on remote with `podman`:
```
[root@rhel8-1 ~]# podman commit --change ENTRYPOINT=/bin/sh --change LABEL=blue=image --message "commit test msg" test-commit image-commited
messages are only compatible with the docker image format (-f docker)
```
This result probably tells us option `-f` can not work too. 

And, I test option `--author` as well, it can work properly.

**Additional environment details (AWS, VirtualBox, physical, etc.):**

### PR DESCRIPTION
**Pull request by jwhonce**
* Some items included in the CLI and currently not supported by the API.

Fixes #1867

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: contrib/python/podman/podman/libs/containers.py [hunk 1]
```diff
@@ -129,21 +126,23 @@ def commit(self,
...
-            author = kwargs.get('author', getpass.getuser())
-        except Exception:  # pylint: disable=broad-except
-            author = ''
+        author = kwargs.get('author', None) or getpass.getuser()
+        change = kwargs.get('change', None) or []
+        message = kwargs.get('message', None) or ''
+        pause = kwargs.get('pause', None) or True
 
-        for c in changes:
+        for c in kwargs['change']:
```
Comment (cevich) [inline contrib/python/podman/podman/libs/containers.py]: Maybe ``for c in kwargs.get('change', [])``?  Not sure if that's correct for this context or not, just responding to the CI test fail.

File: contrib/python/pypodman/pypodman/lib/actions/commit_action.py [hunk 2]
```diff
@@ -80,8 +82,16 @@ def commit(self):
...
             else:
-                ident = ctnr.commit(self.opts['image'][0], **self.opts)
-                print(ident)
+                ident = ctnr.commit(
+                    self.opts['image'][0],
+                    change=self.opts.get('change', None),
+                    message=self.opts.get('message', None),
+                    pause=self.opts['pause'],
+                    author=self.opts.get('author', None),
+                )
```
Comment (cevich) [inline contrib/python/pypodman/pypodman/lib/actions/commit_action.py]: Something special ``pause`` it doesn't get the "gentle" treatment too?

Comment (jwhonce) [reply contrib/python/pypodman/pypodman/lib/actions/commit_action.py]: pause is boolean type: always True or False in this case never None.

File: contrib/python/podman/podman/libs/containers.py (hunk 1 above)
Comment (jwhonce) [reply contrib/python/podman/podman/libs/containers.py]: Too aggressive F&R, should have been 'change'.  The code above will use get() to default change. PR update coming.

**Pull request merged**